src/main/java/org/spongepowered/api/command/parameter/managed/standard/VariableValueParameters.java

```diff
@@ -168,7 +224,7 @@ public static TextBuilder textBuilder() {
          *               without such a prefix
          * @return This builder, for chaining.
          */
-        CatalogedTypeBuilder<T> prefix(String prefix);
+        CatalogedTypeBuilder<T> defaultNamespace(String prefix);
```

Comment:
I would go for `tryNamespace` or something similar because you can have multiple. `defaultNamespace` sounds like you can only define one.